base/api-deployment.yaml

```diff
@@ -222,16 +216,6 @@ spec:
               value: '$(ZENDESK_API_URL)'
             - name: ZENDESK_API_KEY
               value: '$(ZENDESK_API_KEY)'
-            - name: FF_NOTIFICATION_CELERY_PERSISTENCE
-              value: '$(FF_NOTIFICATION_CELERY_PERSISTENCE)'
-            - name: FF_BATCH_INSERTION
-              value: '$(FF_BATCH_INSERTION)'
-            - name: FF_REDIS_BATCH_SAVING
-              value: '$(FF_REDIS_BATCH_SAVING)'
-            - name: FF_CLOUDWATCH_METRICS_ENABLED
```

Comment:
This key isn't mentioned in your PR description, should it have been?  I don't see it as being removed from the .env files either.

Reply:
the cloudwatch metric one shouldnt be removed and is a good catch @andrewleith , will fix it.
@jimleroyer  can you expand on we will disable monitoring?

Reply:
thank you!